options.rb

```diff
@@ -7,7 +7,8 @@
 require_relative 'modules/list'
 require_relative 'modules/save_album'
 require_relative 'modules/save_genre'
-require_relative 'modules/save_game'
```

Comment:
It's possible that during the conflict resolution you accidentally deleted a line crucial for the app. Please add this line back so that the app can run.